testsuite/features/support/commonlib.rb

```diff
@@ -207,12 +207,15 @@ def generate_repository_name(repo_url)
 
 def extract_logs_from_node(node)
   os_family = node.os_family
-  node.run('zypper --non-interactive install tar') if os_family =~ /^opensuse/
-  node.run('journalctl > /var/log/messages', check_errors: false) # Some clients might not support systemd
+  node.run('zypper --non-interactive install tar') if os_family =~ /^opensuse/ && !$is_container_provider
```

Comment:
do we need this if we update the images as in above^^ ?

Reply:
Yes, this is to skip this line in case we are in GH